### DISCUSSION TITLE
Dependance on symfony/symfony

### DISCUSSION THREAD

**Discussion by gilles-g** [lexik/LexikTranslationBundle#428] [General]
Hi @bartmcleod ,

Have you seen that tweet https://twitter.com/fabpot/status/1610635976660893698 

> If you're still using the symfony/symfony package instead of the individual [#Symfony](https://twitter.com/hashtag/Symfony?src=hashtag_click) packages, please, upgrade ASAP.

I think they want to remove this package...But I'm not sure

We use it on the dev deps.

**Comment by bartmcleod** [comment]
@gilles-g I don't think I ever saw this discussion, my bad. I only came here today to bring up something entirely different. I will have a look.

**Reply by bartmcleod** [reply]
We are no longer using symfony/symfony in the dev-deps
